CODEOWNERS

```diff
@@ -0,0 +1 @@
+*	@thanujalk @brionmario @NipuniBhagya @inthirakumaaran @thivi @DimalChandrasiri @jeradrutnam
```

Comment:
Add a new line here

Reply:
since this is read by the bot, i don't think that a new line is necessary.